com.gluon.scenebuilder.yml

```diff
@@ -0,0 +1,69 @@
+app-id: com.gluon.Scenebuilder
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: SceneBuilder
+finish-args:
+  - --require-version=0.10.3
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --socket=ssh-auth
+  - --share=network
+  - --device=all
+  - --filesystem=xdg-config/gtk-3.0
+  - --filesystem=host
```

Comment:
this is a very broad file system permission

Reply:
it should access file system to save file in different path based on user required. I think it should be user space is enough.

Reply:
updated with minimal permissions now.